src/ontodia/viewUtils/connectionsMenu.tsx

```diff
@@ -25,11 +25,27 @@ const ALL_RELATED_ELEMENTS_LINK: FatLinkType = new FatLinkType({
     diagram: null,
 });
 
+export interface PropertySuggestionParams {
+    id: string;
```

Comment:
We should be more explicit here: the name `id` isn't enough here to understand that this is a diagram element id. A better name would be `elementId`.